packages/app-project/stores/User/UserPersonalization/UserProjectPreferences/UserProjectPreferences.js

```diff
@@ -5,16 +5,26 @@ import asyncStates from '@zooniverse/async-states'
 import logToSentry from '@helpers/logger/logToSentry.js'
 import numberString from '@stores/types/numberString'
 
-const Preferences = types
+export const Preferences = types
   .model('Preferences', {
     minicourses: types.maybe(types.frozen()),
     selected_workflow: types.maybe(types.string),
```

Comment:
I think this is also a workflow ID, in which case it should be `numberString` too.